crates/modelardb_compression/src/models/swing.rs

```diff
@@ -627,6 +685,21 @@ mod tests {
             &[42.0, 42.0, 42.8, 42.0, 42.0],
         ))
     }
+    // #[test]
```

Comment:
Leftover code should not be merged into the `main` branch. If code is no longer used it should be deleted. However, in this case it looks like unfinished tests, so it would probably be better to finish implementing the tests instead of just deleting the code.

Reply:
The test was added.